nuxeo-runtime/nuxeo-connect-standalone/src/test/java/org/nuxeo/launcher/connect/TestConnectBroker.java

```diff
@@ -124,9 +131,9 @@ public void handle(String target, HttpServletRequest request, HttpServletRespons
     @Before
     public void beforeEach() throws Exception {
         // set fake Connect connector
-        String addonJSON = FileUtils.readFileToString(new File(testStore, "addon_remote.json"), UTF_8);
-        String hotfixJSON = FileUtils.readFileToString(new File(testStore, "hotfix_remote.json"), UTF_8);
-        String studioJSON = FileUtils.readFileToString(new File(testStore, "studio_remote.json"), UTF_8);
+        String addonJSON = FileUtils.readFileToString(new File(testStore, "addon_remote.json"));
```

Comment:
This method is deprecated, please keep the one with charset.

Reply:
Nice catch, they were added by mistake.